lib/pages/item_request.dart

```diff
@@ -113,34 +111,18 @@ class _ItemRequestState extends State<ItemRequest> {
     );
   }
 
-  Future<void> addLost() {
-    // Calling the collection to add a new user
-    return lost
-        //adding to firebase collection
-        .add({
-      //Data added in the form of a dictionary into the document.
-      'Date Created': Timestamp.now(),
-      'Description': description,
-      'ItemName': category,
-      'Location': GeoPoint(lat, long),
-      'Phone': phone,
-      'Picture': _imgName
-    });
+  Future<void> addLost() async {
+    Item item = Item(id: "", userId: AuthService.instance.user!.uid, timeCreated: Timestamp.now(), description: description, itemName: category, location: GeoPoint(lat, long),phone: phone, picture: _imgName);
+    DocumentReference ref = await FirebaseFirestore.instance.collection('lost').add(item.toFirestore());
+    item.id = ref.id;
+    DataService.instance.lostItems.add(item);
```

Comment:
There should be a better way to manage lost and found items. We stream from FirebaseFirestore for a reason, it allows us to sync with changes in the backend. There should not be a need to add via locally. We can instead stream the collection and update any info needed to be displayed accordingly.

Reply:
Updated the methods to not add locally. Doesn't update automatically but it does if you refresh